CHANGELOG.md

```diff
@@ -18,6 +18,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
                 Makefile.
  * [`fixed`]    Fix uVision compilation warnings (#1295-D: Deprecated
                 declaration of shtc1_sleep - give arg types)
+ * [`changed`]  Change the function name prefix to `shtcx_` for all functions
+                that work with all sensors of the SHTC1 family (SHTC1, SHTW2,
```

Comment:
@rnestler What about the SHTWx series which also use this driver? That's the main reason I remain skeptical of renaming.

Reply:
Yes, that is a bit of a wart. Do we also need something like a decoder ring from marketing to engineering names like for GPUs? https://www.x.org/wiki/RadeonFeature/#index5h2
I guess it may cause more confusion.

Reply:
We could rename it to `shtcx_wx` or similar.